app/assets/images/payment_icons/simpl.svg

```diff
@@ -0,0 +1,17 @@
+<svg width="38" height="24" viewBox="0 0 38 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
The CI checks failed due to incomplete markup.  
`{:message=>"The 'simpl' SVG file should have a single <title> tag"}.`

Please make sure that your SVG markup meets the following [criteria](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#guidelines-for-new-icons).